### ISSUE TITLE
Does not update resources

### ISSUE THREAD

**Issue by xarek1986** [terraform-providers/terraform-provider-selectel#59]
Hello there!
When I'm trying to change project quotas it has no effect

### Terraform Version
Terraform v0.12.3
+ provider.selectel v2.2.0

### Affected Resource(s)
Please list the resources as a list, for example:
- selectel_vpc_project_v2

### Terraform Configuration Files
```hcl
variable "vcpus" {
  default = "4"
}


provider "selectel" {
  token = "${var.sel_token}"
}

resource "selectel_vpc_project_v2" "someproject" {
  name = "someproject"

  quotas {
    resource_name = "compute_cores"
    resource_quotas {
      region = "ru-2"
      zone   = "ru-2b"
      value  = "${var.vcpus}"
    }
  }
}
```

### Expected Behavior
Terraform has to update project quotas

### Actual Behavior
Terraform reported that no changes

>Acquiring state lock. This may take a few moments...
selectel_vpc_project_v2.someproject: Refreshing state... [id=xxxxxxx]

>Apply complete! Resources: 0 added, 0 changed, 0 destroyed.
Releasing state lock. This may take a few moments...

### Steps to Reproduce
1. `terraform appy`: first run
2. `terraform apply -var vcpus=6`: try to update vcpus number
3. change `vcpus` variable in tf file and run `terraform apply`

**Comment by ozerovandrei** [comment]
@xarek1986 thank you for raising this issue.

This bug was fixed in #64.
I will create a new release of the provider as soon as possible.

**Comment by ozerovandrei** [comment]
`2.3.0` version has been released.
